scripts/merge_detections.py

```diff
@@ -64,7 +64,7 @@
     for dataset, dets_file in DETECTION_FILES.items():
         detections_ds_gdf = gpd.read_file(dets_file)    
         detections_ds_gdf[f'dataset'] = dataset
-        detections_gdf = pd.concat([detections_gdf, detections_ds_gdf], axis=0)
+        detections_gdf = pd.concat([detections_gdf, detections_ds_gdf], axis=0).reset_index(drop=True)
```

Comment:
Using the parameter `ignore_index` available with `concat` would be better for readability.